packages/mds-web-sockets/types.ts

```diff
@@ -1,2 +1,2 @@
-export const ENTITY_TYPES = ['EVENTS', 'TELEMETRIES'] as const
-export type ENTITY_TYPE = typeof ENTITY_TYPES[number]
+export const EntityTypes = ['event', 'telemetry'] as const
+export type EntityType = typeof EntityTypes[number]
```

Comment:
I went with PascalCase while updating as it seems to be the standard form?  Objections welcome.

Reply:
I think most other const assertions we have use the UPPER_SNAKE_CASE format.

Reply:
+1